RPIPs/RPIP-46.md

```diff
@@ -1,108 +0,0 @@
-
-This proposal also includes a small set of items for potential future use:
-- Seals that the security council can use to increase the NO share -- this is being used to find a reasonable setting based on the actual market.
-- An allowlist of controllers that may make changes to the settings, which allows for potential automation in the future
-
-## Specification
-1. Inflation settings SHALL be modified to retain inflation to the DAOs and eliminate inflation to NOs
-   1. `rpl.inflation.interval.rate` SHALL be set to `1000040763630249500` (1.5% per year)
-   2. Node Operators (`rocketClaimNode`) allocation SHALL be set to 0%
-   3. pDAO (`rocketClaimDAO`) allocation SHALL be set to 95%
-   4. oDAO (`rocketClaimTrustedNode`) allocation SHALL be set to 5%
-2. There SHALL be the following defined shares with settings: `node_operator_commission_share`, `voter_share`, `surplus_share`
-   1. `node_operator_commission_share`: each NO receives this percentage of the commission from the borrowed ETH on validators they run. Unlike the remainder of the shares, this is _not_ a protocol revenue (ie, it is not socialized).
-   2. `voter_share`: each NO receives a share of revenue based on the vote-eligible RPL staked to their megapool. The overall voter share of revenue is based on the setting, and each NO receives a proportion of that based on `vote_eligible_RPL_in_their_megapool/total_vote_eligible_RPL_in_megapools`.
-   3. `surplus_share`: this share of revenue is used to distribute revenue beyond that used for protocol operation (such as the shares above)
-3. `reth_commission` SHALL be defined as the sum of all defined shares that have settings
-4. `reth_share` SHALL be defined as `100% - reth_commission`
-5. Distributions of revenue from borrowed ETH MUST respect the defined shares
-   1. If shares change between claims, distributions MUST make an effort to account for the different values. For example, a distribution could use a duration-weighted average share. Approximations MAY be used where they significantly reduce complexity and/or costs.
-   2. Legacy minipools are an exception and MAY continue to support earlier distribution methodologies 
-6. `node_operator_commission_share`, `surplus_share`, `increase_no_share_seal_increment`, `increase_no_share_seal_count`, and `allowlisted_controllers` MAY be updated by pDAO vote
-7. `node_operator_commission_share` and `surplus_share`, MAY be updated by an address in the `allowlisted_controllers` array
-   1. This functionality SHALL not be used without a separate pDAO vote to enable a controller and add it to the list
-8. The security council SHALL have a limited-use power to increase the `node_operator_commission_share` by `increase_no_share_seal_increment` and decrease the `surplus_share` by the same amount
```

Comment:
Rather than have this system of seals, does it make more sense to just have the change on a timer? EG security council can move it up or down once every 14 days only, within the predefined min/max. Just seems a lot less complicated than seals and replenishment by pDAO.

Reply:
If not, should be an entry in rationale section to explain maybe. Seems over-engineered as is.

Reply:
Alternatively, if we want to explicitly enable multiple changes in a larger burst, have seals replenish up to a limit once every 2-weeks.

Reply:
Requiring pDAO to manually replenish seems like a tax we can avoid, they should only need to intervene if they disagree with the council's choices, rather than intervene to agree.

Reply:
That's pretty nice, yeah. There's also a setting-based variant without a seal power that was brought up in discord.

Reply:
Put in the setting-based variant. Not married to it per se.